samples/dotnet/KernelHttpServer/SemanticKernelEndpoint.cs

```diff
@@ -97,7 +97,8 @@ public SemanticKernelEndpoint(IMemoryStore memoryStore)
             return await req.CreateResponseWithMessageAsync(HttpStatusCode.BadRequest, "Missing one or more expected HTTP Headers");
         }
 
-        var planner = new SequentialPlanner(kernel);
+        // TODO: Support SequentialPlanner
```

Comment:
Would you please create an issue in the Apps and Services project and reference that issue in this comment? Same for the other TODOs. This will help us track these TODOs better.

Reply:
Done https://github.com/orgs/microsoft/projects/852/views/6?filterQuery=Sequential&pane=issue&itemId=31417114